pkg/webui/console/containers/header/notifications-dropdown.js

```diff
@@ -56,50 +58,65 @@ const NotificationsDropdown = () => {
           />
         </span>
       </div>
-      {dropdownItems.slice(0, 3).map(notification => (
-        <Link
-          to={{
-            pathname: `/notifications/all/${notification.id}`,
-          }}
-          key={notification.id}
-          className={classnames(style.notificationsDropdownLink, 'd-flex')}
-        >
-          <Icon icon="key" className={style.notificationsDropdownLinkIcon} />
-          <div className={style.notificationContainer}>
-            <div className={classnames(style.title, 'fw-bold')}>
-              <Notification.Title
-                data={notification}
-                notificationType={notification.notification_type}
-              />
-            </div>
-            <div
-              className={classnames(
-                notificationStyle.notificationPreviewContent,
-                style.previewContent,
-              )}
-            >
-              <Notification.Preview
-                data={notification}
-                notificationType={notification.notification_type}
-              />
-            </div>
-            <Status pulse={false} status="good" className="mr-cs-xs" />
-            <DateTime.Relative
-              relativeTimeStyle="short"
-              showAbsoluteAfter={2}
-              dateTimeProps={{
-                time: false,
-                dateFormatOptions: { month: '2-digit', day: '2-digit', year: 'numeric' },
+      {dropdownItems && dropdownItems.length === 0 ? (
+        <div className="flex-grow al-center p-vert-ls-m">
+          <Message
+            content={m.noNotifications}
+            className="d-block text-center fs-l fw-bold c-text-neutral-semilight"
+          />
+          <Message
+            content={m.noNotificationsDescription}
+            className="d-block text-center c-text-neutral-light"
+          />
+        </div>
```

Comment:
```suggestion
        <div className="flex-grow al-center p-vert-ls-m">
          <Message
            content={m.noNotifications}
            className="d-block text-center fw-bold c-text-neutral-semilight"
          />
          <Message
            content={m.noNotificationsDescription}
            className="d-block text-center fs-s mb-ls-m c-text-neutral-light"
          />
        </div>
```
Please make sure you apply the correct values from the design since The font sizes were off. It would be good if we were able to avoid review iterations like these.